### ISSUE TITLE
When connected via SSH i just see "Connecting" as the path on my host...?

### ISSUE THREAD

**Issue by lynxus** [Eugeny/tabby#3383]
Version: 1.0.130
Platform: win32 10.0.18363
Plugins: none

So.. For example, you can see below, i connect to a host.. But "COnnecting" stays as the path.
Rather than showing, say /home/gsmart$

So i can issue commands, but the Connecting but stays there?

▫ Connecting
▫ Connecting
▫ Connecting
total 437292
drwxr-xr-x  4 root   root        4096 Jan 27 18:03 ../
□ Connecting

**Comment by lynxus** [comment]
Further debugging has found that this only happens when i use a private key for authentication that requires a password.

If the key doesnt need a password to use then it seems to work fine.

- Workaround , use puttygen to remove password from private key and use that.

**Comment by Eugeny** [comment]
Please capture DevTools console logs at the time of the issue: https://github.com/Eugeny/terminus/wiki/Debug-logs

**Comment by lynxus** [comment]
Interestingly, after a system restart and trying to replicate again, I cant.

Guess its best to close the issue, if it arises again ill grab console output.

**Comment by amajauskas** [comment]
Hello @Eugeny, I am experiencing the same issues as described here, only when SSH key auth is required. 
I was able to capture the log, [click here to download](https://github.com/Eugeny/terminus/files/6407209/terminuslog-1619806107966.log).

I turned on the program, connected to one of my servers, typed a couple of commands like ls, ls -l, and then exit. This is the console output of that session.

Thanks
